index.html

```diff
@@ -0,0 +1,164 @@
+                </ul>
+            </article>
+        </section>  
+
+        <section id="What_you_should_already_know" class="main">
+            <header>What You Should Already Know</header>
+            <article>
+                <p>This guide assumes you have the following basic background:</p>
+                <ul>
+                    <li>A general understanding of the Internet and the World Wide Web (WWW).</li>
+                    <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+                    <li>Some programming experience. If you are new to programming, try one of the tutorials linked on the main page about JavaScript.</li>
+                </ul>
+            </article>
+        </section>  
+
+        <section id="#Java_Script_and_Java" class="main">
+            <header>JavaScript and Java</header>
+            <article>
+                <p>JavaScript and Java are similar in some ways but fundamentally different in some others. The JavaScript language resembles Java but does not have Java's static typing and strong type checking. JavaScript follows most Java expression syntax, naming conventions and basic control-flow constructs which was the reason why it was renamed from LiveScript to JavaScript.</p>
+                <p>In contrast to Java's compile-time system of classes built by declarations, JavaScript supports a runtime system based on a small number of data types representing numeric, Boolean, and string values. JavaScript has a prototype-based object model instead of the more common class-based object model. The prototype-based model provides dynamic inheritance; that is, what is inherited can vary for individual objects. JavaScript also supports functions without any special declarative requirements. Functions can be properties of objects, executing as loosely typed methods.</p>
+                <p>JavaScript is a very free-form language compared to Java. You do not have to declare all variables, classes, and methods. You do not have to be concerned with whether methods are public, private, or protected, and you do not have to implement interfaces. Variables, parameters, and function return types are not explicitly typed.</p>
+            </article>
+        </section>  
+
+        <section id="#Hello_World" class="main">
+            <header>Hello World</header>
+            <article>
+                <p>To get started with writing JavaScript, open the Scratchpad and write your first "Hello world" JavaScript code:</p>
+                <div class="code-box">
+                <code>function greetMe(yourName) { alert("Hello " + yourName); }
+                    <br>greetMe("World");</code>
+                </div>
+               <p>Select the code in the pad and hit Ctrl+R to watch it unfold in your browser!</p> 
+            </article>
+        </section>  
```

Comment:
Kindly use an article tag instead of section tag because this part is independent self-contained content.